src/main/java/uk/gov/hmcts/reform/sendletter/entity/LetterRepository.java

```diff
@@ -25,6 +25,14 @@
         @Param("before") LocalDateTime before
     );
 
+    @Query("select l from Letter l where l.status not in ('Posted', 'Aborted')"
+        + " and l.createdAt < :createdBefore and l.type <> '"
```

Comment:
SLA really applies to `sentToPrintAt` value as it is _exact_ figure when external service is aware of letter supposed to be printed. Plus downtime window - shifts the business day really.

Reply:
A stale letter is now a letter that hasn't been posted, even though it's been 2 business days since receiving the request. It's not just about the external service's SLA anymore - we want to be alerted whenever for any reason a letter wasn't posted, regardless of where it got stuck.

As for the date, it's not the repository's concern (it receives it as a parameter), but we should definitely take another look at https://tools.hmcts.net/jira/browse/BPS-621 and make it clear when a business day starts - 00:00 or 17:00.